### PR TITLE
updates from the comments

### PR DESCRIPTION
**Pull request by nixxus1030**
resolution for points 87 86 17

### CODE REVIEW

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 1]
```diff
@@ -3057,16 +3053,17 @@ components:
       properties:
         minimalPrice:
           $ref: '#/components/schemas/CurrencyPriceDef'
-        overallServiceClass:
-          $ref: '#/components/schemas/ServiceClassDef'
+        overallServiceClassId:
```
Comment (CGantert345) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:3056]: we need more details on the interpretation of "overall". As i understood this is only an indication which can optionally be used for connecting offers from sources.

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 2]
```diff
@@ -2949,26 +2950,21 @@ components:
...
-        tripRef:
-          type: string
-          format: url
-          example: 'http://some.provider/trip/2345'
-        trip:
-          $ref: '#/components/schemas/Trip'
-        requestedSection:
-          $ref: '#/components/schemas/TravelSectionDef'
+        tripSearch:
+          oneOf:
```
Comment (CGantert345) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:2958]: we should clear the oneOff construct. As the generated code is too bad we decided to have the oneOff only in cmments and not as part of the schema

**Review by CGantert345 — CHANGES_REQUESTED** [review]

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml (hunk 1 above)
Comment (nixxus1030) [reply IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:3056]: Overall XXX is the result of "adding up" all the values of underlying offerParts. I described how these addition should take place in the documentation

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 3]
```diff
@@ -3205,9 +3205,9 @@ components:
         country:
           $ref: '#/components/schemas/CountryDef'
         amount:
-          description: amount
+          description: amount in cents
```
[the remark this answers is not in the document]
Comment (nixxus1030) [reply IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:3208]: I don't really understand how the currency has an impact here: the point is that the number  is a number in cents, or if you prefer in 1/100th of currency units. I actually looked for best practice on how to represent amounts in JSON and came up (a bit to my surprise tbh) with this model (integer number expressing the amount in 1./100ths of units) the most common recommendation

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml (hunk 2 above)
Comment (nixxus1030) [reply IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:2958]: I understand the point and can agree but IMO this is something to be discussed with the group so it is clear to everyone, and then to be applied to the whole specifications (not a single oneOf structure should then remain)

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 4]
```diff
@@ -3540,6 +3537,12 @@ components:
             oneOf:
               - $ref: '#/components/schemas/PassengerRef'
               - $ref: '#/components/schemas/Passenger'
+        providerOrderReference:
```
[the remark this answers is not in the document]
Comment (nixxus1030) [reply IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml]: I changed all references to an "orderReference " with "bookingReference", which is something we know

**Review by schlpbch — APPROVED** [review]

**Pull request merged**